Snippets/Core/Core_6/Core_6.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.ServiceFabric.Services" Version="2.*" />
-    <PackageReference Include="NServiceBus" Version="6.*" />
+    <PackageReference Include="NServiceBus" Version="6.5-*" />
```

Comment:
This is going to mean there will need to be a prerelease.txt in the snippets folder, but then that will also mean we'll end displaying a prerelease warning on all v6 pages.

I think we might need to split this out in some other way. @DavidBoike Any ideas?

Reply:
I think there has to be a Core_6.5 folder temporarily.

Reply:
I’ll give it a try next week.